sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -51,23 +53,26 @@ We guarantee that all client instance methods are thread-safe and independent of
 
 ## Examples
 
-- [Query logs](#query-logs)
-- [Query logs as model](#query-logs-as-model)
-- [Query logs as primitive](#query-logs-as-primitive)
-- [Batch query](#batch-query)
-- [Query dynamic table](#query-dynamic-table)
-- [Increase query timeout](#increase-query-timeout)
+* [Query logs](#query-logs)
```

Comment:
We use `-` in other readmes. Is `*` better in some way?

Reply:
Let's standardize on `-` then. I'll update it.